src/components/tracked-controls.js

```diff
@@ -21,6 +21,7 @@ module.exports.Component = registerComponent('tracked-controls', {
     id: {type: 'string', default: ''},
     idPrefix: {type: 'string', default: ''},
     rotationOffset: {default: 0},
+    hand: {type: 'string'},
```

Comment:
Yes, I think we should remove it all together. This should not be a choice make by the application